include/pidfd-utils.h

```diff
@@ -31,6 +31,13 @@ static inline int pidfd_open(pid_t pid, unsigned int flags)
 }
 #  endif
 
+#  ifndef HAVE_PIDFD_GETFD
+static inline int pidfd_getfd(int pidfd, int targetfd, unsigned int flags)
+{
+	return syscall(SYS_pidfd_getfd, pidfd, targetfd flags);
```

Comment:
`,` between `targetfd` and `flags` is missed.

Reply:
See https://github.com/util-linux/util-linux/pull/3202/commits/2a4c540255edb938042ccc6db2cfa1155f6e14d5